keyboards/index.php

```diff
@@ -47,14 +73,14 @@
 
 <div class='<?= $embed == 'none' ? '' : 'embed embed-'.$embed ?>'>
 
-  <h2 class="red underline"><a href='/keyboards'>Keyboard Search</a></h2>
+  <h2 class="red underline"><a href='/keyboards'><?= $keyboardIndexStrings['Keyboard Search'] ?></a></h2>
 
   <div id='search-box'>
     <form method='get' action='/keyboards' name='f'>
-      <label for="search-q">Keyboard search:</label><input id="search-q" type="text" placeholder="Enter language or keyboard" name="q"
+      <label for="search-q"><?= Locale::_s($keyboardIndexStrings['Keyboard search%s'], ":") ?></label><input id="search-q" type="text" placeholder="<?= $keyboardIndexStrings['Enter language or keyboard'] ?>" name="q"
```

Comment:
Why have you split the `:` out from the string? Here and other places? Seems to make the strings much more complicated than they need to be

Reply:
ok, I've consolidated punctuation back into the strings in edd7430134e7881bd1a532ee3fbe321bdf58e908